test/cpp/jit/torch_python_test.cpp

```diff
@@ -34,29 +34,30 @@ void testEvalModeForLoadedModule() {
   AT_ASSERT(module.attr("dropout").toModule().is_training());
 }
 
-void testSerializationInterop() {
-  if (isSandcastle()) {
-    // The module file to load is not generated in Sandcastle
-    return;
-  }
+// TODO: this test never ran before and is broken.
```

Comment:
Why didn't this run?

Reply:
It was copied into this file like that two years ago: https://github.com/pytorch/pytorch/pull/44795. I commented it out with the same comment that the test invocation was commented out with. I didn't realize it was that old when I kept it in, opening an issue makes sense here. #79275

It looks like when it was copied two years ago it was already dead code.